### PR TITLE
Added interpolation search

### LINKED ISSUE #24: Package name missing in InterpolationSearch.java
Please fix : Package name missing in InterpolationSearch.java

### PR DESCRIPTION
**Pull request by Shwetannaa**

### CODE REVIEW

File: javaCode/src/searchAlgos/HeapSearch.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
+// Java program to implement interpolation
```
Comment (soumyachan) [inline javaCode/src/searchAlgos/HeapSearch.java]: Package declaration is missing #24

**Review by soumyachan — CHANGES_REQUESTED** [review]
Please fix : Package declaration missing in this file : : #24

**Review by shri-platypus — CHANGES_REQUESTED** [review]
Please resolve issue #24

**Review by shri-platypus — APPROVED** [review]
Issue #24 fixed now.

**Review by soumyachan — APPROVED** [review]
Issue is fixed.

**Pull request merged**